### PR TITLE
Upgrade to Pulumi v1.0.0

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -169,11 +169,12 @@ func Provider() tfbridge.ProviderInfo {
 			"packet_operating_system":    {Tok: makeDataSource(mainMod, "getOperatingSystem")},
 			"packet_precreated_ip_block": {Tok: makeDataSource(mainMod, "getPrecreatedIpBlock")},
 			"packet_spot_market_price":   {Tok: makeDataSource(mainMod, "getSpotMarketPrice")},
+			"packet_device":              {Tok: makeDataSource(mainMod, "getDevice")},
```
Comment (pgavlin) [inline resources.go:172]: Bugfix?

Comment (stack72) [reply resources.go:172]: Yes, this was missing from the SDK generation and I found it now and added it in

Comment (pgavlin) [reply resources.go:172]: kk, might add that to the CL

**Review by pgavlin — APPROVED** [review]

File: resources.go (hunk 1 above)
Comment (stack72) [reply resources.go:172]: 👍

**Pull request merged**